lib/index.js

```diff
@@ -1,55 +1,82 @@
-var zlib = require('zlib'),
-    each = require('async').each,
-    mm = require('micromatch');
-
-function plugin(options) {
-    const defaultOptions = {
-        src: "**/*.+(html|css|js|json|xml|svg|txt)",
-        gzip: {
-            level: 6
-        },
-        overwrite: false
-    };
+const zlib = require('zlib'),
+      bytes = require('bytes');
```

Comment:
I think this package might not be required. Since this code only uses the number -> string code path and there's no need to use different formatting options, it reduces to very little code that we could define ourselves (compare https://github.com/visionmedia/bytes.js/blob/9ddc13b6c66e0cb293616fba246e05db4b6cef4d/index.js#L84)

Reply:
Sure, do. But builds should sensibly display B -> GB or convert everything to MB perhaps